Actions/AL-Go-Helper.ps1

```diff
@@ -4,6 +4,7 @@ Param(
 
 $gitHubHelperPath = Join-Path $PSScriptRoot 'Github-Helper.psm1'
 $readSettingsModule = Join-Path $PSScriptRoot '.Modules/ReadSettings.psm1'
+$debugLoggingModule = Join-Path $PSScriptRoot '.Modules/DebugLogHelper.psm1'
```

Comment:
I think you need to add this in localdevenv too 
https://github.com/microsoft/AL-Go/blob/c245bb14451539f2c7a3137e9dd58c0594a5073d/Templates/AppSource%20App/.AL-Go/localDevEnv.ps1#L58